Tests/Marketplace/copy_and_upload_packs.py

```diff
@@ -247,6 +247,20 @@ def verify_copy(successful_packs, pc_successful_packs_dict):
     assert not not_uploaded and not mistakenly_uploaded, error_str
 
 
+def check_if_need_to_upload(pc_successful_packs_dict, pc_failed_packs_dict):
+    """ If the two dicts are empty then no upload was done in Prepare Content step, so we need to skip uploading
+
+    Args:
+        pc_successful_packs_dict: The successful packs dict
+        pc_failed_packs_dict: The failed packs dict
+
+    """
+    if not pc_successful_packs_dict and not pc_failed_packs_dict:
```

Comment:
Why do we want to continue the upload if there were packs that failed to upload in previous step?

Reply:
Because we will handle the failed packs in the packs loop, this is by design